frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx

```diff
@@ -123,6 +123,7 @@ export const CreateOBCPage: React.FC<CreateOBCPageProps> = (props) => {
                 name="storageClass"
                 hideClassName="co-required"
                 filter={showOnlyObcScs}
+                id="sc-dropdown"
```

Comment:
you are adding it for tests, so add test prefix.
will be easy to track later.

Reply:
I don't think we should add test prefix in plugin code.

Reply:
Agreed with @bipuladh. We should not add.